tests/foreman_api_test.py

```diff
@@ -38,6 +39,7 @@ def test_foreman_lifecycle_environment(lifecycle_environment):
 
 
 def test_foreman_content_view(content_view, yum_repository, foremanapi):
+    time.sleep(10)
```

Comment:
This sleep helped. This is not a deterministic solution, and I am investigating a better way to properly find and wait on the right set of tasks. In the meantime I think this is an acceptable solution to avoid constant CI re-runs.